reference/configuration/framework.rst

```diff
@@ -53,13 +53,11 @@ out all the application users.
 handle_all_throwables
 ~~~~~~~~~~~~~~~~~~~~~
 
-**type**: ``boolean`` **default**: ``false``
+**type**: ``boolean`` **default**: ``true``
 
-If set to ``true``, the Symfony kernel will catch all ``\Throwable`` exceptions
+When set to ``true``, the Symfony kernel will catch all ``\Throwable`` exceptions
 thrown by the application and will turn them into HTTP responses.
 
-Starting from Symfony 7.0, the default value of this option will be ``true``.
```

Comment:
Did you mean to target the `7.0` branch?

Reply:
Yeah, and yes my target branch wasn't the right one ;)